From 51da7f36e5a3f72f2001a72e4c679146c2df013b Mon Sep 17 00:00:00 2001
From: auto commit <unknown@unknown>
Date: Fri, 24 Oct 2008 12:23:13 -0700
Subject: [PATCH] cavium: Enable HCD for USB if Cavium.

If board is Cavium, then select USB_ARCH_HAS_HCD by default.

Signed-off-by: Tomaso Paoletti <tpaoletti@caviumnetworks.com>
Signed-off-by: Paul Gortmaker <Paul.Gortmaker@windriver.com>
---
 drivers/usb/Kconfig |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/drivers/usb/Kconfig b/drivers/usb/Kconfig
index 3265afe..73fffa1 100644
--- a/drivers/usb/Kconfig
+++ b/drivers/usb/Kconfig
@@ -22,6 +22,7 @@ config USB_ARCH_HAS_HCD
 	default y if PCMCIA && !M32R			# sl811_cs
 	default y if ARM				# SL-811
 	default y if SUPERH				# r8a66597-hcd
+	default y if CPU_CAVIUM_OCTEON
 	default PCI
 
 # many non-PCI SOC chips embed OHCI
-- 
1.5.5.1

